chia/rpc/wallet_rpc_api.py

```diff
@@ -2978,13 +2980,17 @@ async def get_farmed_amount(self, request) -> EndpointResult:
             # .get_farming_rewards() above queries for only confirmed records.  This
             # could be hinted by making TransactionRecord generic but streamable can't
             # handle that presently.  Existing code would have raised an exception
-            # anyways if this were to fail and we already have an assert below.
+            # anyway if this were to fail and we already have an assert below.
             assert height is not None
+            timestamp = await self.service.get_timestamp_for_height(height)
```

Comment:
you only need the timestamp on line 2993. it would be more efficient to only ask for it in that case